src/main/java/transforms/CombineDateTimeTransform.java

```diff
@@ -95,8 +92,8 @@ public Object apply( Map<String, String> row ) {
             return null;
         }
         LocalDateTime dateTime = LocalDateTime.of( date, time );
-        TimeZone tz = this.timezone;
-        OffsetDateTime out = dateTime.atZone( tz.toZoneId() ).toOffsetDateTime();
+        Optional<TimeZone> tz = this.timezone;
+        OffsetDateTime out = dateTime.atZone( tz.orElse(Constants.DEFAULT_TIMEZONE).toZoneId() ).toOffsetDateTime();
```

Comment:
Aw man, that's correct, since the checkTimezone is effectively replacing that part.
However, for transparency, do we really want to put in a default timezone if it's not gonna be (nor should be) used?

Reply:
I see, it's anyway used in JavaDateTimeHelper.  I'll undo those changes.

Reply:
ok done